src/metadata_validator.py

```diff
@@ -74,8 +74,9 @@ def metadataValidate(configs, job_queue, mongo_dao):
                             validator.submission[VALIDATION_ENDED] = validation_end_at
                         mongo_dao.set_submission_validation_status(validator.submission, None, status, None, None)
                     elif data.get(SQS_TYPE) == TYPE_CROSS_SUBMISSION and submission_id:
+                        data_commons = data.get(DATA_COMMONS)
                         validator = CrossSubmissionValidator(mongo_dao)
-                        status = validator.validate(submission_id)
+                        status = validator.validate(submission_id, data_commons)
```

Comment:
It seems data_commons will be passed in as **scope** parameter in validator.validate function, and it only accepts "New", any other values will be treated the same. 
Besides, I think only cross-validation needs to be changed for the bug.

Reply:
The validator object in this context is a CrossSubmissionValidator object. The validate function of CrossSubmissionValidator uses the data_commons as part of this bug fix update